app/models/concerns/solr_holdings.rb

```diff
@@ -29,7 +29,15 @@ def browseable_spines
                              browseable_schemes = %w[LC DEWEY ALPHANUM]
                              browseable_items = items.select { |v| v.shelfkey.present? && browseable_schemes.include?(v.callnumber_type) }
 
-                             browseable_items.uniq(&:shelfkey).map do |v|
+                             representative_items = browseable_items.group_by(&:truncated_callnumber).map do |_base_call_number, items|
+                               if items.one?
+                                 items.first
+                               else
+                                 items.min_by(&:full_shelfkey)
```

Comment:
Do we need the conditional?  Can't we use `min_by` for all cases?

Reply:
Functionally, yes, although I thought being explicit was clearer 🤷‍♂️